docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+In addition to the above, here is a sampling of the decisions you will need to make before deploying components:
+
+* **Certificate Authority configuration**.
+
+  As part of the overall decisions you have to make about your peers (how many, how many on each channel, and so on) and about your ordering service (how many nodes, who will own them), you also have to decide on how the CAs for your organization will be deployed. Do you want to use TLS? Then you will need a TLS CA, and it will need to be deployed before your enrollment CA. We'll discuss this more in [Step 3: Set up your CAs](#step-3-set-up-your-cas).
+
+* **Use Organizational Units or not?**
+
+  Some organizations might find it necessary to establish Organizational Units to create a separation between certain identities and MSPs created by a single CA.
+
+* **Database type.**
+
+  Some channels in a network might require all data to be modeled in a way [CouchDB](couchdb_as_state_database.html) can understand, while other networks, prioritizing speed, might decide that all peers will use LevelDB.
+
+* **Channels and private data.**
+
+  Some networks might decide that [channels](channels.html) are the best way to ensure privacy and isolation for certain transactions. Others might decide that a single channel, along with [private data collections](private-data/private-data.html), better serves their need for privacy.
+
+* **Container orchestration.**
+
+  Different users might also make different decisions about their container orchestration, creating separate containers for their peer process, logging for the peer, CouchDB, gRPC communications, and chaincode, while other users might decide to combine some of these processes.
+
+* **External chaincode.**
+
+  The new [external chaincode service](cc_service.html) will also be implemented by some users and not others.
+
+However and wherever your components are deployed, you will need a high degree of expertise in your management system of choice (such as Kubernetes) in order to efficiently operate your network. Similarly, the structure of the network must be designed to fit the business use case and any relevant laws and regulations government of the industry in which the network will be designed to function.
+
+This deployment guide will not go through every iteration and potential network configuration, but does give common guidelines and rules to consider.
+
+## Step two: Set up a cluster for your resources
+
+Generally speaking, Fabric is agnostic to the method used to deploy and manage it. It is possible, for example, to deploy and manage a peer from a laptop. For a number of reasons, this is likely to be unadvisable, but there is nothing in Fabric that prohibits it.
```

Comment:
The concept of 'this is determined by the use-case' is a common point. 
Could this be made prominent at the beginning, then the rest of the guide can focus on the features of that use case that will drive the decisions?

Reply:
This is the real trick of this whole doc, and a delicate balancing act. "Step one" is meant to introduce some of these things at a high level and talk about how use cases will influence those decisions. But that section could be improved.